serveradmin/resources/management/commands/clear_migration_log.py

```diff
@@ -0,0 +1,49 @@
+"""Serveradmin - igvm_migration_log cleanup
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+from django.core.management.base import BaseCommand
+from adminapi.dataset import Query
+from adminapi.filters import Not, Empty
+from datetime import datetime
+
+class Command(BaseCommand):
+    help = 'Check for old igvm_migration_log entries and deletes them'
+
+    def handle(self, *args, **options):
+        """Entry point for command
+
+        get all hypervisors which have an entry in igvm_migration_log
+        and delete all entries which are from the previous day.
+
+        """
+
+        all_hypervisor = Query({'igvm_migration_log': Not(Empty())},
+                           ['igvm_migration_log', 'hostname'])
+
+        for hypervisor in all_hypervisor:
+
+            for entry in hypervisor['igvm_migration_log']:
+                timestamp = entry.split(' ')[0]
+
+                dt_object_sa = datetime.fromtimestamp(float(timestamp))
+                day_sa = dt_object_sa.day
+                month_sa = dt_object_sa.month
+
+                current_month = datetime.now().month
+                current_day = datetime.now().day
+
+                if (day_sa - current_day == 0 and
+                        current_month - month_sa == 0):
```

Comment:
What about `year`?

Reply:
The timedelta class could be useful here: https://docs.python.org/3/library/datetime.html#timedelta-objects

Reply:
integrated timedelta class.